packages/build-utils/test/unit.test.ts

```diff
@@ -576,6 +576,28 @@ it(
   ms('1m')
 );
 
+it('should return cliType npm when no lockfile is present', async () => {
+  const originalRepoLockfilePath = path.join(
+    __dirname,
+    '..',
+    '..',
+    '..',
+    'pnpm-lock.yaml'
+  );
+  const originalRepoLockfileData = await fs.readFile(originalRepoLockfilePath);
+  await fs.remove(originalRepoLockfilePath);
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (blocking):** ‏why are we referring to a lockfile that doesn't exist?

Am I missing something about how this test works?

Reply:
I think you misunderstand.

The function we're testing here is `scanParentDirs()` and it will walk up the dir tree until it finds a lockfile. Since the test fixture doesn't have a lockfile, it keeps walking up the tree until it finds the monorepo root lockfile. So it incorrectly thinks the test fixture has a lockfile.

We could update `scanParentDirs()` to accept a new parameter that would stop walking the tree at a certain point but instead I just delete and restore the monorepo's lockfile so it won't be detected.